assets/src/components/lateView.tsx

```diff
@@ -24,20 +38,28 @@ const LateView = (): ReactElement<HTMLElement> => {
   const lateBusThreshold = 60 * 15
   const missingLogonThreshold = 60 * 45
 
+  const withinMissingLogonThreshold = (ghost: Ghost) =>
+    currentTime - (ghost.scheduledLogonTime as number) <= missingLogonThreshold
```

Comment:
How sure are you about doing `as number` here? Should we add a case specifically for when the value is null?

Reply:
Never mind, just scrolled down a couple of lines. Carry on. 🙂